Foundation/NSPropertyList.swift

```diff
@@ -117,6 +138,8 @@ internal func _expensivePropertyListConversion(input : AnyObject) -> Any {
         return true
     } else if input === kCFBooleanFalse {
         return false
+    } else if CFGetTypeID(input) == _CFKeyedArchiverUIDGetTypeID() {
```

Comment:
@parkera do we want to extract this out? or will it be ok leaving it here since it is an internal class type?